frontend/src/App.vue

```diff
@@ -24,113 +34,129 @@
 </template>
 
 <script>
-import { mapState, mapActions, mapGetters } from 'vuex';
-
-import Header from './components/Header.vue';
-import Refresh from './components/Refresh.vue';
-import Status from './components/Status.vue';
-import Sidebar from './components/Sidebar.vue';
-import store, { LANGUAGE } from './store';
-
-import { IS_USER_LOGGED_IN } from './store/modules/http/getters';
-
-import { SYNC } from './store/constants';
-import { FETCH_USER } from './store/actions';
-
-export default {
-  name: 'App',
-  store,
-  components: {
-    Refresh,
-    Header,
-    Status,
-    Sidebar,
-  },
-  data() {
-    return {
-      showMenu: true,
-    };
-  },
-  computed: {
-    ...mapState([LANGUAGE]),
-    ...mapGetters('http', [IS_USER_LOGGED_IN]),
-  },
-  methods: {
-    ...mapActions({
-      sync: SYNC,
-      fetchUser: FETCH_USER,
-    }),
-  },
-  created() {
-    if (this.language) {
-      /* eslint-disable-next-line no-underscore-dangle */
-      this._i18n.locale = this.language;
-    }
-    if (this[IS_USER_LOGGED_IN]) {
-      this.sync();
-      this.fetchUser();
-    }
-  },
-};
+  import { mapState, mapActions, mapGetters } from 'vuex';
```

Comment:
I agree that it makes sense to indent the code in `<script>` and `<style>` but we are not doing it in this project (at least for now) so if you'd like to see that, we can introduce a formatter with such rule, but generally I'd stay away from reformatting the code against existing (although unwritten) rules.